framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -330,6 +333,106 @@ public void processClassTree(ClassTree classTree) {
         super.visitClass(classTree, null);
     }
 
+    /**
+     * Check that the field invariant meets the following requirements:
```

Comment:
I did this at with a reference to the manual section at the bottom of the Javadoc.